setup.cfg

```diff
@@ -68,7 +69,8 @@ max-line-length = 120
 exclude =
     build,dist,.git,.idea,.cache,.tox,.eggs,
     ./src/vorta/views/dark/collection_rc.py,
-    ./src/vorta/views/light/collection_rc.py
+    ./src/vorta/views/light/collection_rc.py,
```

Comment:
These files don't exist anymore, so these lines should be deleted

Reply:
Right. Good catch.